src/test/resources/extension.js

```diff
@@ -35,8 +35,9 @@ function executeScript(scriptEvent) {
     log.warn('hello from script with event: ' + scriptEvent);
     extensionLogger.logInfo(scriptEvent);//logs into the extension_log table
     if(scriptEvent === 'INVOICE_GENERATION') {
-	return {
-		invoiceNumber: 'blabla'
-	};
+        return {
+            invoiceNumber: 'blabla'
+        };
     }
+    return null;
```

Comment:
Added `return null` here, making this test pass. Thank you for helping us help you! We are ready for the next iteration. 👍